driver.go

```diff
@@ -767,11 +767,11 @@ func checkIsValidType(v driver.Value) bool {
 	case uint:
 	case []uint:
 	case *uint:
-	case *[]uint:
```

Comment:
@egonelbre Can we keep existing one add the new ones, not sure if it will be a breaking change removing the *[]uint

Reply:
Yeah, we should not remove `*[]uint` if it has previously been included in a released version, as that would be a breaking change (however unlikely it is that someone is using it).

There are also some test errors. I don't see anything in the PR description or the comments that would indicate that those are expected. Would you mind taking a look at that?

Reply:
Sure, it would be possible to keep them, although, they were broken before -- so I'm quite confident they weren't being used before. :)

It would also imply adding things like `*[]int64` for symmetry.

Reply:
As for the test failures, yeah, interesting that they weren't failing for me locally.

Reply:
> Sure, it would be possible to keep them, although, they were broken before -- so I'm quite confident they weren't being used before. :)

If they indeed did not work, then it is OK to remove them.

Reply:
The implementation is trivial, so I'll add it.

Reply:
Ah, the failing test was a different one, which was checking the ConvertParam implementation.

Reply:
done